torchci/rockset/metrics/__sql/master_jobs_red_avg.sql

```diff
@@ -20,7 +20,7 @@ FROM
             job.name != 'ciflow_should_run'
             AND job.name != 'generate-test-matrix'
             AND workflow.event != 'workflow_run' -- Filter out worflow_run-triggered jobs, which have nothing to do with the SHA
-            AND push.ref = 'refs/heads/master'
+            AND push.ref IN ('refs/heads/master', 'refs/heads/main')
```

Comment:
This one also looks slightly different--pls rebase as well.

fbb27c782608d52d

Reply:
@janeyx99 This one is failing saying that it doesn't exist in the metrics namespace. Is that on rockset? Because it exists in the `rockset/metrics/__sql` folder

Reply:
Turns out I couldn't read properly and the hash should be 29b138b4454f2a63